### PR TITLE
Removed jquery dependency from navbar

### PR DESCRIPTION
**Pull request by mathieubrun**
Had a bit of fun with slideToggle removal !

### CODE REVIEW

**Comment by mathieubrun** [comment]
⚠️  I did not update the minified file

**Comment by Sylhare** [comment]
Awesome job! 🎉 

It seems to be working fine on my side. I am impressed!
How did you do, I see you have been playing with the css to make smoother transition. Can you brief me on it?

File: _sass/includes/_navbar.scss [hunk 1]
```diff
@@ -113,8 +114,25 @@
 
         ul {
             width: 100%;
-            display: none;
-            height: auto;
+            overflow: hidden;
+
+            transition: font-size .25s,
```
Comment (mathieubrun) [inline _sass/includes/_navbar.scss]: used transition here

**Comment by mathieubrun** [comment]
Thanks, as I updated the template for my needs its normal to share

**Pull request merged**